apis/extension/elastic_quota.go

```diff
@@ -38,6 +38,7 @@ const (
 	AnnotationSharedWeight = QuotaKoordinatorPrefix + "/shared-weight"
 	AnnotationRuntime      = QuotaKoordinatorPrefix + "/runtime"
 	AnnotationRequest      = QuotaKoordinatorPrefix + "/request"
+	AnnotationQuotaLabel   = QuotaKoordinatorPrefix + "/quota-label"
```

Comment:
What is the function of this new annotation?  Please update the documentation if necessary.

Reply:
fixed